atomics/T1518.001/T1518.001.yaml

```diff
@@ -34,19 +34,45 @@ atomic_tests:
       get-process | ?{$_.Description -like "*defender*"}
       get-process | ?{$_.Description -like "*cylance*"}
     name: powershell
-- name: Security Software Discovery - ps
+- name: Security Software Discovery - ps (macOS)
   auto_generated_guid: ba62ce11-e820-485f-9c17-6f3c857cd840
   description: |
     Methods to identify Security Software on an endpoint
-    when sucessfully executed, command shell  is going to display AV software it is running( Little snitch or carbon black ).
+    when sucessfully executed, command shell  is going to display AV/Security software it is running.
   supported_platforms:
-  - linux
   - macos
   executor:
     command: |
       ps -ef | grep Little\ Snitch | grep -v grep
       ps aux | grep CbOsxSensorService
       ps aux | grep falcond
+      ps aux | grep nessusd
+      ps aux | grep santad
+      ps aux | grep CbDefense
+      ps aux | grep td-agent
+      ps aux | grep packetbeat
+      ps aux | grep filebeat
+      ps aux | grep auditbeat
+      ps aux | grep osqueryd
+      ps aux | grep BlockBlock
+      ps aux | grep LuLu
+    name: sh
+- name: Security Software Discovery - ps (Linux)
+  description: |
+    Methods to identify Security Software on an endpoint
+    when sucessfully executed, command shell  is going to display AV/Security software it is running.
+  supported_platforms:
+  - linux
+  executor:
+    command: |
+      ps aux | grep falcond
```

Comment:
I'm not sure what is more common. But this is certainly more efficient, on both Linux and MacOS. Again if you feel this is more what is intended let me know and ill approve and merge.

`ps aux | egrep 'falcond|nessusd|cbagentd|td-agent|packetbeat|filebeat|auditbeat|osqueryd'`

Reply:
hey @rc-grey! yes, that will work as well, not so much different from a behavioral perspective, but certainly more efficient. will change it right now